### ISSUE TITLE
From PDF to text with revelant heading

### ISSUE THREAD

**Issue by ammaarahmad1999** [kermitt2/grobid#751]
How can I convert from PDF to text retaining all headings and subheadings and removing images from it

**Comment by kermitt2** [comment]
Hello @ammaarahmad1999 !

I don't understand your question because you describe the normal usage of grobid. Could you elaborate?

**Comment by ammaarahmad1999** [comment]
Hello
I have a folder of pdf files. I want to convert it to text using grobid.
I have 2 problems?
1. Do I need to do it 1 by 1
2. I am able to convert a single pdf from PDF to TEI/XML format but I don't
know how to convert TEI/XML to plain text files with all headings present.
Can you help in these issues

**Comment by kermitt2** [comment]
> Do I need to do it 1 by 1

No, please read the documentation https://grobid.readthedocs.io ! 

> 2. I am able to convert a single pdf from PDF to TEI/XML format but I don't know how to convert TEI/XML to plain text files with all headings present.

You have to learn the basics of XML. There are plenty of tutorials online and tools to retrieve the text in the section of interests, depending on your programming or non-programming user profile.
